README.md

```diff
@@ -229,39 +241,56 @@ The `source.properties` defines add-on properties used for the built image, whic
       <minor>0</minor>
       <micro>0</micro>
     </revision>
-    <display-name>ProfusionAddOn, Android 29, rev 1</display-name>
+    <display-name>ProfusionAddOn, Android 30, rev 51</display-name>
     <uses-license ref="license-CC939D3F"/>
   </localPackage>
 </ns2:repository>
 ```
 The `package.xml` file makes it easier for Android Studio to import the add-on.
 
 ##### `sdk_addon_stub_defs.txt`
+
 ```code
 +com.profusion.helloworld.*
 ```
 The `sdk_addon_stub_defs.txt` file contains the rules for public APIs.
 
 For these tests, I was using `lunch aosp_x86_64-eng`, so I decided to add the reference to `profusion_sdk_addon.mk` in the `build/make/target/product/aosp_x86_64.mk`, but you may add it to whichever device you are using. Simply add:
-```code
+
+```make
 $(call inherit-product, $(SRC_TARGET_DIR)/product/profusion_sdk_addon/profusion_sdk_addon.mk)
 ```
 
-### Building
+### Building the SDK addon
 The SDK Add-on is all set. Now we only have to build it. Notice, again, that I'm using `aosp_x86_64`:
-```code
-$ make PRODUCT-aosp_x86_64-sdk_addon
+
+```bash
+. build/envsetup.sh
+export TARGET=profusion_sdk_addon
+lunch aosp_x86_64-eng
+make -j 4 sdk_addon
 ```
 
 When the compilation succeeds, the SDK Add-on `.zip` will be in `out/host/linux-x86/sdk_addon/profusion_sdk_addon-eng.user-linux-x86.zip`
 
+### Building Android
+
+To build Android containing the library that we want to supply, do
+
+```bash
+# . build/envsetup.sh if you are doing it on a new shell
+lunch aosp_x86_64-eng
+m
+```
+
 ## Adding the SDK Add-on to Android Studio
 
 Create a path on your Android SDK for the add-ons, then extract the contents of the add-on `.zip` file. You will need to rename the directory using the names defined in the "localPackage path" in the `package.xml`
-```code
-$ mkdir -p /path/to/Sdk/add-ons
-$ unzip out/host/linux-x86/sdk_addon/profusion_sdk_addon-eng.user-linux-x86.zip -d /path/to/Sdk/add-ons/
-$ mv /path/to/Sdk/add-ons/profusion_sdk_addon-eng.user-linux-x86/ /path/to/Sdk/add-ons/addon-profusionaddon-profusion-29/
+
+```bash
+mkdir -p /path/to/Sdk/add-ons
+unzip out/host/linux-x86/sdk_addon/profusion_sdk_addon-eng.user-linux-x86.zip -d /path/to/Sdk/add-ons/
```

Comment:
Maybe indicating that `user` should be edited or switching to `${whoami}` or `$USER` is a good idea